packages/@o3r/eslint-config/src/rules/typescript/unicorn.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { TSESLint } from '@typescript-eslint/utils';
+// @ts-expect-error -- Typings will be available once migrate to v55
```

Comment:
Maybe you can add a `todo` to make sure to remove it when going throw them (if forgotten in the v55 migration PRs)